paasta_tools/envoy_tools.py

```diff
@@ -120,6 +103,36 @@ def get_casper_endpoints(
     return frozenset(casper_endpoints)
 
 
+def get_backends_from_eds(service: str, envoy_eds_path: str) -> List[Tuple[str, int]]:
+    """Returns a list of backends for a given namespace. Casper backends are also returned (if present).
+
+    :param service: Return backends for this namespace
+    :param envoy_eds_path: path where EDS yaml files are stored
+    :returns backends: A list of touples representing the backends for
+                       the requested service
+    """
+    backends = []
+    eds_file_for_service = f"{envoy_eds_path}/{service}/{service}.yaml"
+
+    if os.access(eds_file_for_service, os.R_OK):
+        with open(f"{envoy_eds_path}/{service}/{service}.yaml") as f:
+            eds_yaml = yaml.safe_load(f)
+            for resource in eds_yaml.get("resources", []):
+                endpoints = resource.get("endpoints", [])
```

Comment:
I don't think you need the default arg if you're checking for None below anyway (get will return None by default)

Reply:
yeah you are right